.travis.yml

```diff
@@ -9,7 +9,7 @@ before_install:
 - git checkout master
 - git checkout -
 script:
-- COVERAGE=y npm test
+- COVERAGE=y NODE_ENV=production npm test
```

Comment:
In addition to this change, please document this new behavior in the [build section](https://github.com/mozilla/web-ext/blob/master/CONTRIBUTING.md#build-web-ext). I would suggest adding a new paragraph that says:

By default, `npm run build` creates a development build of `web-ext`. To create a production build, use the `NODE_ENV` variable like this:

    NODE_ENV=production npm run build